labman/gui/templates/library_pooling.html

```diff
@@ -75,15 +78,41 @@
    * @param {Array} concentrations A 2D array of floating point values
    * representing the concentrations of a plate.
    * @param {Array} blanks A 2D array of well names.
+   * @param {Float} defaultClipping A number representing the default clipping
+   * value for the heatmap. Depends on the processing stage and the type of
+   * data being processed.
    *
    */
-  function createHeatmap(plateId, concentrations, blanks, names) {
-    var container = $('#pool-results-' + plateId);
-    var heatmap = $('<div id="heatmap-' + plateId + '"></div>');
-    var histogram = $('<div id="histogram-' + plateId + '"></div>');
+  function createHeatmap(plateId, concentrations, blanks, names,
```

Comment:
We want to also have this code in the quantify interface. 

Should we be locating this function in a different, centralized file and importing?